test/unit/kubernetes-deploy/google_friendly_config_test.rb

```diff
@@ -14,7 +14,7 @@ def teardown
   def test_auth_use_default_gcp_success
     config = KubernetesDeploy::KubeclientBuilder::GoogleFriendlyConfig.new(kubeconfig, "")
 
-    stub_request(:post, 'https://www.googleapis.com/oauth2/v3/token')
+    stub_request(:post, 'https://www.googleapis.com/oauth2/v4/token')
```

Comment:
Since this is a gem, the Gemfile.lock isn't committed to the repo. Our CI bundle installs from scratch every time, so it'll upgrade as far as the Gemfile allows it to when new versions become available. Because of this, CI just started using v0.4.0 of kubeclient, which is what caused this difference. Alternatively, we could pin kubeclient more strictly in the gemspec.